CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ### Changed
 
--   BC-589 - BC-651 - Change auto deployment to a reusable workflow
+-   BC-589, BC-651 - Change auto deployment to a reusable workflow
```

Comment:
Please let it be with the ' - ' in it between the two ticket numbers
```suggestion
-   BC-589 - BC-651 - Change auto deployment to a reusable workflow
```

Reply:
sure. fixed